libs/beacon/src/beacon_setup_service.cpp

```diff
@@ -726,7 +729,7 @@ BeaconSetupService::State BeaconSetupService::OnDryRun()
 
     for (auto const &key_and_count : dry_run_public_keys_)
     {
-      if (key_and_count.second >= beacon_->manager.polynomial_degree())
+      if (key_and_count.second >= QualSize())
```

Comment:
You swapped threshold for qualsize here? isn't the qual size variable?

Reply:
This is what David proposed for the dry run. That we set a minimum for the qual size, which is higher than the signature threshold, and impose that we only succeed in the dry run if we meet at least minimum qual size number of people with the same group public key. QualSize() is not variable, it is some minimum we impose whereas the actual size of qual is indeed variable.